features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -30,7 +30,11 @@
 And(/^the authentication for the "([^"]*)" is established$/) do |package_name|
   json_keystone = proposal("keystone")
 
-  auth_node = json_keystone["deployment"]["keystone"]["elements"]["keystone-server"].first
+  if json_keystone["attributes"]["keystone"]["ha"]["enabled"] == true
+    auth_node = json_keystone["deployment"]["keystone"]["elements_expanded"]["keystone-server"].first
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)